test/__mocks__/session.mock.ts

```diff
@@ -711,8 +743,26 @@ const hasSoldLandKey = '0';
 const isSecureRegisterKey = '0';
 
 export const TRUST: Trust = {
+  trust_id: "",
+  trust_name: "name of trust",
+  creation_date_day: "31",
+  creation_date_month: "12",
+  creation_date_year: "1999",
+  unable_to_obtain_all_trust_info: "No"
+};
+
+export const TRUST_NO_NAME: Trust = {
   trust_id: "",
   trust_name: "",
+  creation_date_day: "31",
+  creation_date_month: "12",
+  creation_date_year: "1999",
+  unable_to_obtain_all_trust_info: "No"
+};
```

Comment:
Perhaps worth adding a mock and test when just one of the date field component fields is empty/undefined.